src/foam/u2/wizard/controllers/IncrementalWizardController.js

```diff
@@ -146,6 +146,7 @@ foam.CLASS({
     async function setFirstPosition() {
```

Comment:
Move this func to step wizard controller?

Reply:
technically `next` should actually be moved to IncrementalWizardController, but that will break a lot of things right now. It's a big refactor where we haven't had time to do a big refactor in a while.